index.js

```diff
@@ -16,7 +16,9 @@ function exportToJsonString(idbDatabase, cb) {
     );
     transaction.onerror = (event) => cb(event, null);
 
-    Array.from(idbDatabase.objectStoreNames).forEach((storeName) => {
+    const objectStoreNames = Array.from(idbDatabase.objectStoreNames);
```

Comment:
change Array.from(idbDatabase.objectStoreNames) to Set(idbDatabase.objectStoreNames) avoid duplications store names

Reply:
Done